storage/innobase/xtrabackup/src/ds_local.cc

```diff
@@ -144,6 +159,11 @@ static int local_write_sparse(ds_file_t *file, const void *buf, size_t len,
       return 1;
     }
 
+    if (is_fallocate_punch_hole_supported()) {
```

Comment:
Can you check if we can use server method os_file_punch_hole

Reply:
@rahulmalik87 - Fixed the commit message (Since there were no code changes, I force-pushed it).

Regarding the os_file_punch_hole. I tried that and also using is_punch_hole_supported from innodb part. However, that would mean that we will need to start compiling the whole InnoDB suite for xbstream and xbcrypt.
Currently, both tools are standalone.

Reply:
@rahulmalik87 regarding 2.4 - sparse file support at xbstream / incremental is not supported.